Warsaw_Tram/MapViewController.swift

```diff
@@ -46,18 +51,22 @@ class MapViewController: UIViewController, AlertHelperProtocol {
     }
     
     private func displayTramsOnMap() {
-        tramViewModel.getTramsData({ (trams) in
-            for tram in trams where tram.status == "RUNNING" {
-                dispatch_async(dispatch_get_main_queue(), {
+        let priority = DISPATCH_QUEUE_PRIORITY_DEFAULT
```

Comment:
It doesn't looks good. To many if-s in controller. I'd rather move this to ViewModel, it is more what to display on map not how to display on map.